### ISSUE TITLE
train.py

### ISSUE THREAD

**Issue by Zhuysheng** [chaoyuaw/pytorch-coviar#15]
i follow the GETTING_STARTED.md, but  when run train.py :
Augmentation scales: [1, 0.875, 0.75, 0.66]
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream

**Comment by chaoyuaw** [comment]
Hi @Zhuysheng , are you able to provide more details, or a minimal code that reproduces the error? Thanks!

**Comment by Zhuysheng** [comment]
@chaoyuaw first, i use reencode.sh to create the mpeg4_videos ,
then run   
python train.py --lr 0.0003 --batch-size 40 --arch resnet152 \
 	--data-name ucf101 --representation iframe \
 	--data-root data/ucf101/mpeg4_videos \
 	--train-list data/datalists/ucf101_split1_train.txt \
 	--test-list data/datalists/ucf101_split1_test.txt \
 	--model-prefix ucf101_iframe_model \
 	--lr-steps 55 110 165  --epochs 220 \
 	--gpus 0 1

after print the net, it shows could not open input stream,

Training arguments:
        data_name: ucf101
        data_root: data/ucf101/mpeg4_videos
        train_list: data/datalists/ucf101_split1_train.txt
        test_list: /home/zys/pytorch-coviar/data/datalists/ucf101_split1_test.txt
        representation: iframe
        arch: resnet152
        num_segments: 3
        no_accumulation: False
        epochs: 220
        batch_size: 40
        lr: 0.0003
        lr_steps: [55.0, 110.0, 165.0]
        lr_decay: 0.1
        weight_decay: 0.0001
        eval_freq: 5
        workers: 8
        model_prefix: ucf101_iframe_model
        gpus: [0, 1]

Initializing model:
    base model:         resnet152.
    input_representation:     iframe.
    num_class:          101.
    num_segments:       3.

Model(
  (base_model): ResNet(
    (conv1): Conv2d(3, 64, kernel_size=(7, 7), stride=(2, 2), padding=(3, 3), bias=False)
   ........
           )
Augmentation scales: [1, 0.875, 0.75, 0.66]
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
Could not open input stream
^Z
[2]+  Stopped

**Comment by FelixZhang7** [comment]
@Zhuysheng hey,i met same error,have you figure it out yet ?

**Comment by Zhuysheng** [comment]
@chaoyuaw @FelixZhang7 sorry, i havent solve it  ,any idea?

**Comment by Tylerjoe** [comment]
maybe you should check the video used for train or test.
you can print the video name in data_loader/coviar_data_loader.c to get which video can not be opened

**Comment by Pengxiao-Wang** [comment]
@Zhuysheng  hey,i met same error,have you figure it out yet ?
